tests_metricflow/snapshots/test_cli.py/str/BigQuery/test_saved_query__cli_output.txt

```diff
@@ -1,11 +1,11 @@
-| metric_time__day    |   listing__capacity_latest |   bookings |   instant_bookings |
-|:--------------------|---------------------------:|-----------:|-------------------:|
-| 2019-12-01 00:00:00 |                          5 |          1 |                  0 |
-| 2019-12-18 00:00:00 |                          4 |          4 |                  2 |
-| 2019-12-19 00:00:00 |                          4 |          6 |                  6 |
-| 2019-12-19 00:00:00 |                          5 |          2 |                  0 |
-| 2019-12-20 00:00:00 |                          5 |          2 |                  0 |
-| 2020-01-01 00:00:00 |                          4 |          2 |                  1 |
-| 2020-01-02 00:00:00 |                          4 |          3 |                  3 |
-| 2020-01-02 00:00:00 |                          5 |          1 |                  0 |
-| 2020-01-03 00:00:00 |                          5 |          1 |                  0 |
+metric_time__day      listing__capacity_latest    bookings    instant_bookings
+------------------  --------------------------  ----------  ------------------
+2019-12-01                                   5           1                   0
```

Comment:
Oh interesting. Is that because the datetime/dateutil objects don't format in 0 values for time?

Reply:
This is actually due to a case in the text formatting logic. In retrospect, let me remove that.